lib/iris/tests/stock.py

```diff
@@ -35,6 +35,53 @@
 from iris.coord_systems import GeogCS, RotatedGeogCS
 
 
+def geodetic(shape, with_bounds=True, xlim=None, ylim=None, circular=False):
+    """
+    Return a global lat-lon cube of specified shape.
```

Comment:
This could do with adding "2D", clarifying "global", and describing `xlim`, `ylim`, `circular`. (Although this needs to be considered in the light of a merger with `lat_lon_cube`.)